src/Api/ProductMediaFileApi.php

```diff
@@ -18,6 +19,7 @@ class ProductMediaFileApi implements MediaFileApiInterface
     const MEDIA_FILES_PATH = 'api/rest/v1/media-files';
     const MEDIA_FILE_PATH = 'api/rest/v1/media-files/%s';
     const MEDIA_FILE_DOWNLOAD_PATH = 'api/rest/v1/media-files/%s/download';
+    const MEDIA_FILE_URI_CODE_REGEX = '~/api/rest/v1/media\-files/(.*)$~';
```

Comment:
Sorry I don't understand what you mean by "pattern". For me the value of this constant is a pattern.